guides/advanced_guides/field_properties.md

```diff
@@ -0,0 +1,36 @@
+# Field properties
+
+Every document field can have either or both or none of the following properties that can be modified in the [settings](/references/settings.md):
+
+- Searchable
+- Displayed
+
+## Searchable fields
+
+Searchable fields are fields whose attribute is stored in the [searchable-attributes](/references/searchable_attributes.md) list.
+
+The content of a searchable field is used by MeiliSearch to **assess the relevancy of a document**.
+
+Therefore, when a field's attribute is not in the searchable-attribute list, the field will be ignored from searches but will remain stored in the server.
+
+By default, all field attributes are added to the searchable-attributes list. If a new document is ingested and one of its fields is missing from other documents, the latter will be automatically added to the searchable-attributes list. [This behavior can be modified](/references/accept_new_fields.md).
+
+This list can be restricted to a specific set of attributes that you can select in the settings. That way, you are able to specify which fields should be ignored by MeiliSearch from searches.
+
+## Displayed attributes
+
+Displayed fields are fields whose attribute is stored in the [displayed-attributes](/references/displayed_attributes.md) list.
```

Comment:
To not confuse with the storing in Meilisearch?
```suggestion
Displayed fields are fields whose attribute is present in the [displayed-attributes](/references/displayed_attributes.md) list.
```

Reply:
Yes, right!